.github/workflows/ci.yml

```diff
@@ -52,11 +52,6 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Set Python 3.7
-        uses: actions/setup-python@v4
-        with:
-          python-version: 3.7
-          architecture: ${{ matrix.arch }}
```

Comment:
Thanks, but please keep it for verifying PDM can still **manage** Python 3.7. We just drop Python 3.7 as a working version that PDM can be **installed** with.

Reply:
What is the python support policy for this project? Python 3.7 is end of life

Reply:
We tend to extend that lifetime for management. But PDM itself no longer works on 3.7 right?

Reply:
i'm not saying it doesn't make sense to support older versions, i'm just wondering what the policy is?

- why support 3.7 and not 3.6?
- if 3.7 support won't be dropped now, then when?

Would be good to have a policy documented

Reply:
It is difficult to have a clear boundary for the minimum supported version, as it depends on several factors:

1. The syntax used by the **in-process** scripts. This is maintained by PDM, but it only needs little effort to keep syntax compatibility.
2. The toolchain support, such as build backends. This is not controlled by PDM, some tools are still supporting up to Python 3.6, in this case users are able to run their projects on older Python versions. However some other tools may have an aggressive policy so users working on this tools are not able to use old Python versions.

In the CI matrix we are justing running a simple integration against Python 3.7, it doesn't mean we  only supports up to Python 3.7, or not supporting, for the above reasons.